src/models/users.js

```diff
@@ -10,6 +9,7 @@ let usersSchema = new mongoose.Schema({
   collegeId: Number,
   regNo: Number,
   teams: [ Number ],
+  isAdmin: Boolean,
```

Comment:
Why do we need `isAdmin`?
We can just check the `type` field and see if the user is a admin or not.

These are the diff. types of users (we need to remove judge & participants from this and add a type for our support team):
https://github.com/manipalutsav/backend/blob/021e2b70a8e933d94123d4a06de956d2aef062b6/src/utils/constants.js#L1-L7

Reply:
Okay. that was a mistake, i forgot to remove it before commit.

Reply:
The resulting values.